.github/workflows/linux.yaml

```diff
@@ -90,7 +90,7 @@ jobs:
       - name: Install Build Dependencies
         run: |
           sudo apt-get update
-          sudo apt-get install gcc autoconf automake make libsdl2-dev libsdl2-image-dev libsdl2-ttf-dev libsdl2-mixer-dev
+          sudo apt-get install gcc autoconf automake make libsdl2-dev libsdl2-image-dev libsdl2-ttf-dev libsdl2-mixer-dev ninja-build
```

Comment:
I don't think it's necessary to have the SDL2 version built with both autotools and CMake here.  Remove the dependencies that are only needed for the autotools build and remove the "Build" step at lines 98-102 that uses autotools.